src/components/MemoDetail.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="backdrop" @click.self='closeDetail'>
+    <div class="modal">
+      <h2>Memo Detail</h2>
+      <label>
+        <input placeholder="Title" v-model='this.memo.title'>
+      </label>
+      <label>
+        <textarea placeholder="Content" v-model='this.memo.content'></textarea>
+      </label>
+      <button @click='editMemo()'>Edit</button>
+      <button @click='deleteMemo()'>Delete</button>
```

Comment:
引数なしで method を呼び出すのであれば `()` はなくても良いかもです。

```suggestion
      <button @click='editMemo'>Edit</button>
      <button @click='deleteMemo'>Delete</button>
```